src/common/config/env.validation.ts

```diff
@@ -53,8 +45,25 @@ const toBoolean = (value: any): boolean => {
   }
 };
 
-@injectable()
-export class Environment {
+export class EnvironmentVariables {
+  @IsEnum(Environment)
+  NODE_ENV: Environment = Environment.development;
+
+  @IsNumber()
+  @Min(1)
```

Comment:
I suggest to set the minimum value to 1025 to avoid using privileged ports range.

Reply:
done